vagrant/.env

```diff
@@ -6,5 +6,7 @@ MACHINE1_MAC=08:00:27:9e:f5:3a
 
 # https://github.com/tinkerbell/charts/pkgs/container/charts%2Fstack
 HELM_CHART_VERSION=0.4.3
-KUBECTL_VERSION=1.28.3
+KUBECTL_VERSION=1.27.12
```

Comment:
We're going backwards?

Reply:
Should we even have a `.env` here? Should it be `.env.example`?

Reply:
yeah, because the k3d version is actually 1.27.

Reply:
the `.env` is not an example. its used when we run `vagrant up`.

Reply:
Right, but do we expect it to change from 1 env to the other? That's typically how .env is used.

Reply:
yes and no. users can change these values but the name doesnt have any particular significance. its just a file to hold values. We could change it.